src/auth/index.js

```diff
@@ -116,27 +116,15 @@ var AuthenticationClient = function(options) {
  * @return {Object}   Object containing client information.
  */
 AuthenticationClient.prototype.getClientInfo = function() {
-  var clientInfo = {
+  return {
     name: 'node-auth0',
     version: pkg.version,
-    dependencies: [],
-    environment: [
+    env: [
```

Comment:
`env` should not be an array, should be an object. [See the RFC](https://auth0team.atlassian.net/wiki/spaces/DXSDK/pages/111968442/SDK+Telemetry+RFC+ACCEPTED#SDKTelemetryRFC[ACCEPTED]-ProposedDesign) 👈  Auth0 internal link

Reply:
good catch